cmd/stack_import.go

```diff
@@ -106,6 +106,15 @@ func newStackImportCmd() *cobra.Command {
 					}
 				}
 			}
+			// Validate the stack. If --force was passed, issue an error if validation fails. Otherwise, issue a warning.
+			if err := snapshot.VerifyIntegrity(); err != nil {
+				msg := fmt.Sprintf("deployment file contains errors: %v", err)
```

Comment:
Is “deployment file” a term we use?  Our CLI has `Pulumi state delete` and friends - so I’ve assumed this is now something we officially call a “state file”.  

Probably worth auditing some CLI docs around this too (in particular for import/export which predated the “state” naming).

Reply:
I'll change this to `state file`.